### PR TITLE
Added some Canadian Universities

### PR DESCRIPTION
**Pull request by mynameiseileen**

### CODE REVIEW

File: src/main/resources/CanadianUni.json [hunk 1]
```diff
@@ -0,0 +1,364 @@
+{
+    "canadianUniversities": [
+        {
+            "name": "Brock University",
+            "country": "Canada",
+            "address": "1812 Sir Isaac Brock Way, St. Catharines, ON, L2S 3A1",
+            "website": "https://brocku.ca/humanities/philosophy/faculty-and-staff/y sta"
```
Comment (OmriHarary) [inline src/main/resources/CanadianUni.json:7]: I think this url is wrong

Comment (mynameiseileen) [reply src/main/resources/CanadianUni.json]: oops my bad, just fixed it

File: src/main/resources/CanadianUni.json [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+            "country": "Canada",
+            "address": "",
+            "website": ""
+        },
+        {
+            "name": "",
+            "country": "Canada",
+            "address": "",
+            "website": ""
+        }
```
Comment (OmriHarary) [inline src/main/resources/CanadianUni.json:164]: Missing a comma

Comment (mynameiseileen) [reply src/main/resources/CanadianUni.json]: fixed!

File: src/main/resources/CanadianUni.json [hunk 3]
```diff
@@ -0,0 +1,364 @@
+{
+    "canadianUniversities": [
```
Comment (OmriHarary) [inline src/main/resources/CanadianUni.json:2]:
```suggestion
[
```
The list can be the top-level element of the file, I don't think we should nest it another level down like this. Just one less step for the code to do.
Also means removing L1 and L364 and reformatting all the indentation.

Comment (mynameiseileen) [reply src/main/resources/CanadianUni.json:2]: Will do

**Pull request merged**